app/controllers/teams/users_controller.rb

```diff
@@ -1,5 +1,29 @@
 class Teams::UsersController < AuthenticatedController
-   before_action -> { authorize team, policy_class: TeamUsersPolicy }
+    before_action -> { authorize team, policy_class: TeamUsersPolicy }
+
+    def new
+      @user = User.new
+    end
+  
+    def create
+      add_email = user_params.require(:email).downcase
+      @user = User.where(email: add_email).first || User.new(email: add_email)
+      if team.users.include?(@user)
+        @user = User.new
+        flash[:error] = I18n.t('teams.users.create.already_member', email: add_email)
+        render :new
```

Comment:
All of these `render :new` could be removed in the fail scenarios and moved to the bottom of the method, if `and return` is added to the pass method.